submissions/asaMitaka/dom/script.js

```diff
@@ -0,0 +1,224 @@
+        `,
+        class: 'colledge'
+    },
+    {
+        title: 'Late Registration',
+        dataName: '2',
+        description: [
+            "Wake Up Mr. West", "Heard 'Em Say", "Touch the Sky", "Gold Digger", "Skit #1", "Drive Slow", "My Way Home", "Crack Music", "Roses",
+            "Bring Me Down", "Addiction", "Skit #2", "Diamonds From Sierra Leone (Remix)", "We Major", "Skit #3", "Hey Mama", "Celebration", "Skit #4",
+            "Gone", "Diamonds from Sierra Leone", "Back to Basics", "We Can Make It Better", "Late"
+        ],
+        img: 'https://upload.wikimedia.org/wikipedia/en/thumb/f/f4/Late_registration_cd_cover.jpg/220px-Late_registration_cd_cover.jpg',
+        alt: 'Late Registration',
+        source: `
+            <iframe style="border-radius:12px" src="https://open.spotify.com/embed/album/5ll74bqtkcXlKE7wwkMq4g?utm_source=generator" width="100%" height="80" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>
+        `,
+        class: 'late'
+    },
+    {
+        title: 'Graduation',
+        dataName: '3',
+        description: [
+            "Good Morning", "Champion", "Stronger", "I Wonder", "Good Life", "Can't Tell Me Nothing", "Barry Bonds", "Drunk and Hot Girls", "Flashing Lights",
+            "Everything I Am", "The Glory", "Homecoming", "Big Brother", "Good Night"
+        ],
+        img: 'https://upload.wikimedia.org/wikipedia/en/thumb/7/70/Graduation_%28album%29.jpg/220px-Graduation_%28album%29.jpg',
+        alt: 'Graduation',
+        source: `
+            <iframe style="border-radius:12px" src="https://open.spotify.com/embed/album/4SZko61aMnmgvNhfhgTuD3?utm_source=generator" width="100%" height="80" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>
+        `,
+        class: 'graduation'
+    },
+    {
+        title: '808s & Heartbreak',
+        dataName: '4',
+        description: [
+            "Say You Will", "Welcome to Heartbreak", "Heartless", "Amazing", "Love Lockdown", "Paranoid", "RoboCop", "Street Lights", "Bad News",
+            "See You in My Nightmares", "Coldest Winter", "Pinocchio Story"
+        ],
+        img: 'https://upload.wikimedia.org/wikipedia/en/thumb/f/f1/808s_%26_Heartbreak.png/220px-808s_%26_Heartbreak.png',
+        alt: '808andHeartbreak',
+        source: `
+            <iframe style="border-radius:12px" src="https://open.spotify.com/embed/album/3WFTGIO6E3Xh4paEOBY9OU?utm_source=generator" width="100%" height="80" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>
```

Comment:
In each element of the array, you have the same iframe, the difference is only in the link, transfer the iframe to the function and leave only the URL here